src/Spec2-Core/SpAbstractValidation.class.st

```diff
@@ -0,0 +1,38 @@
+"
+I represent a validation to apply on a presenter.
+I hold the logic to validate the presenter and provide a message if the validation fails.
+If you want to provide a custom validation, subclass me.
+"
+Class {
+	#name : #SpAbstractValidation,
```

Comment:
Please implement a class side method #isAbstract to return true.

There is now even a refactoring for that:

![image](https://user-images.githubusercontent.com/5980033/145555731-8c2028ff-fc72-43e4-b1f3-5ce1721e7788.png)

Reply:
Thanks. I was not aware of it.